clients/drcachesim/tracer/raw2trace.h

```diff
@@ -191,12 +263,16 @@ class raw2trace_t {
     unread_from_thread_file(uint tidx, offline_entry_t *dest, size_t count);
     bool
     thread_file_at_eof(uint tidx);
+    const std::vector<module_t> &
```

Comment:
suggest: blank line to separate to make this more readable

Reply:
done